Documentation/operations/upgrade.rst

```diff
@@ -307,7 +307,10 @@ Annotations:
 1.16 Upgrade Notes
 ------------------
 
-* To be determined during v1.16 development.
+* Cilium Envoy daemon set is now enabled by default. If you want to preserve existing
```

Comment:
Is this the case for all upgrades, or only for new installs?

Reply:
It will be for all upgrade pre 1.16.

Reply:
I think it would be good to have something like this:
```suggestion
* Cilium Envoy DaemonSet is now enabled by default, and existing in-container installs will be changed to DaemonSet mode unless specifically opted out of. This can be done by disabling it manually...
```

Just a stronger callout so that people who do read the changelog understand the change.

Reply:
ah thanks, let me make one update for the above.